lib/autoprefixer.js

```diff
@@ -61,8 +69,8 @@ module.exports = function () {
 
             if (res.statusCode === 304) {
                 upstreamETag = res.getHeader('ETag');
-                if (upstreamETag && !(/-autoprefixer"$/.test(upstreamETag))) {
-                    res.setHeader('ETag', upstreamETag.replace(/"$/, '-autoprefixer"'));
+                if (upstreamETag && !(/-autoprefixer\[[a-f0-9]{32}\]"$/.test(upstreamETag))) {
```

Comment:
Not related to the change -- but I can't come up with a case where the other middleware would issue an `ETag` containing `-autoprefixer[...]`, so it seems a bit weird to check for that here.